frontend/src/javascript/prototypes/Array.js

```diff
@@ -74,3 +74,11 @@ Array.prototype.prevId = function (id) {
     const index = this.findIndexForId(id);
     return index !== 0 ? this[index - 1].id : null;
 };
+
+Array.prototype.arrayMin = function () {
```

Comment:
Skoro to już metoda w Array to niepotrzebne w nazwie "array". Zmieniłbym na  minValue

Reply:
Albo nawet "getMinValue", bo jak wiadomo z prezentacji na Kole, funkcje powinny być czasownikami